ks-svarut/src/main/java/no/difi/meldingsutveksling/ks/EDUCoreConverterImpl.java

```diff
@@ -0,0 +1,22 @@
+package no.difi.meldingsutveksling.ks;
+
+import no.difi.meldingsutveksling.core.EDUCore;
+import no.difi.meldingsutveksling.ks.mapping.Handler;
+import no.difi.meldingsutveksling.ks.mapping.HandlerFactory;
+
+import java.util.List;
+
+public class EDUCoreConverterImpl implements EDUCoreConverter {
+    HandlerFactory handlerFactory;
+
+    public EDUCoreConverterImpl(HandlerFactory handlerFactory) {
+        this.handlerFactory = handlerFactory;
+    }
+
+    public Forsendelse convert(EDUCore domainMessage) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1161)